Lib/bdb.py

```diff
@@ -323,6 +438,10 @@ def clear_all_file_breaks(self, filename):
         del self.breaks[filename]
```

Comment:
Clarification: add 'return None' at the end of the function ;-).   I missed 'set_break' and any others because of the folding; thank you for checking.  The relevant PEP8 section begins "Be consistent in return statements."  It is standard to omit 'return None' at the end and leave it implicit when there are no other return statements.

For 'if cond: statement', PEP8 says both 'Rather not:' and 'While sometimes it's okay'.  I think we should leave them alone.